repository/tests/integration_test.rs

```diff
@@ -23,6 +23,7 @@ async fn basic_1() {
     let peers = SharedKnownPeers::new_static(peers);
 
     let server_node_dir = create_temp_dir();
+    let server_node_dir_clone = server_node_dir.clone();
```

Comment:
Why do we need this?

Reply:
for line 48 due to line 41

Reply:
Then clone for the async block at 40, right before it.

```
a
a a_clone(removed) 
```
is better than

```
a a_clone
a(removed) a_clone
```
right?